### PR TITLE
[SNG-DASH-2352] Socket details style fixes

### PR DESCRIPTION
**Pull request by powerczuk**
**DoD:**
- remove black line between examples and code switcher 🆗 
- extend the examples section 🆗 
- change the background for parameter names 🆗 
- fix the install custom socket button position ⛔️ - Install button is deleted. We don't need to install a socket from API Explorer View

screen shot 2017-01-03 at 16 22 27

### CODE REVIEW

File: src/apps/CustomSockets/CustomSockets.js [hunk 1]
```diff
@@ -2,9 +2,6 @@ import React from 'react';
 import Reflux from 'reflux';
 import Helmet from 'react-helmet';
 
-// Utils
-import { DialogsMixin } from '../../mixins';
-
 // Stores and Actions
```
Comment (Flixow) [inline src/apps/CustomSockets/CustomSockets.js]: Do we really need it?

File: src/apps/CustomSockets/CustomSockets.js [hunk 2]
```diff
@@ -14,13 +11,11 @@ import { Container, InnerToolbar } from '../../common/';
 
 // Local components
```
Comment (Flixow) [inline src/apps/CustomSockets/CustomSockets.js]: As above

File: src/apps/SocketsRegistry/DetailsView/SocketDetailInfoBar.js [hunk 3]
```diff
@@ -48,7 +46,7 @@ class SocketsRegistryInfoBar extends Component {
       sidebarContent: {
         left: 0,
         position: 'absolute',
-        top: 300
+        top: 188
```
Comment (Flixow) [inline src/apps/SocketsRegistry/DetailsView/SocketDetailInfoBar.js]: 230. "General Info" is invisible now.

**Comment by Flixow** [comment]
On my mind right section with code samples is too big. Any opinions?

File: src/apps/CustomSockets/CustomSockets.js (hunk 1 above)
Comment (powerczuk) [inline src/apps/CustomSockets/CustomSockets.js]: 👍

File: src/apps/CustomSockets/CustomSockets.js (hunk 2 above)
Comment (powerczuk) [inline src/apps/CustomSockets/CustomSockets.js]: 👍

File: src/apps/SocketsRegistry/DetailsView/SocketDetailInfoBar.js (hunk 3 above)
Comment (powerczuk) [inline src/apps/SocketsRegistry/DetailsView/SocketDetailInfoBar.js]: Yup, I know but @wpasinski is removing whole socket search bar in his task and we'll adjust sidebar position later.

**Comment by powerczuk** [comment]
@adamwardecki what do you think about width of the code samples column ?

**Comment by tomsonpl** [comment]
I think it should be of the same size as the middle section.

**Comment by tomsonpl** [comment]
LGTM now

File: src/apps/CustomSockets/CustomSockets.js (hunk 1 above)
Comment (Flixow) [inline src/apps/CustomSockets/CustomSockets.js]: We still don't need this

File: src/apps/CustomSockets/CustomSockets.js (hunk 2 above)
Comment (Flixow) [inline src/apps/CustomSockets/CustomSockets.js]: Still as above

**Pull request merged**